ide/jdt/test/src/test/java/com/ibm/wala/cast/java/test/JDTJavaTest.java

```diff
@@ -1,82 +0,0 @@
-/*
- * Copyright (c) 2013 IBM Corporation.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- *
- * Contributors:
- *     IBM Corporation - initial API and implementation
- */
-package com.ibm.wala.cast.java.test;
-
-import static org.assertj.core.api.Assertions.fail;
-
-import com.ibm.wala.cast.java.client.JDTJavaSourceAnalysisEngine;
-import com.ibm.wala.cast.java.ipa.callgraph.JavaSourceAnalysisScope;
-import com.ibm.wala.client.AbstractAnalysisEngine;
-import com.ibm.wala.core.tests.callGraph.CallGraphTestUtil;
-import com.ibm.wala.ide.tests.util.EclipseTestUtil.ZippedProjectData;
-import com.ibm.wala.ipa.callgraph.CallGraphBuilder;
-import com.ibm.wala.ipa.callgraph.Entrypoint;
-import com.ibm.wala.ipa.callgraph.impl.Util;
-import com.ibm.wala.ipa.callgraph.propagation.InstanceKey;
-import com.ibm.wala.ipa.cha.IClassHierarchy;
-import com.ibm.wala.util.io.TemporaryFile;
-import java.io.File;
-import java.io.IOException;
-import java.util.Collection;
-import java.util.List;
-
-public abstract class JDTJavaTest extends IRTests {
```

Comment:
I think this used to be used when we were actually running Eclipse Plugin tests by loading them under Maven with Tycho.  But we don't do that anymore, so I think it's ok to delete this one.